recipe/meta.yaml

```diff
@@ -49,4 +49,6 @@ about:
 
 extra:
   recipe-maintainers:
-    - jp
+    - kujaku11
+    - kkappler
+    - lheagy
```

Comment:
This reverts an accidental change [here]( https://github.com/conda-forge/mth5-feedstock/commit/ca8cf948fa01bfa3602eb738c8fd2faaac146c67#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR50-R52 )